docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -1251,19 +1368,23 @@ spring:
   cloud:
     gateway:
       routes:
-      - id: removerequestparameter_route
+      - id: removeresponseheader_route
         uri: https://example.org
         filters:
-        - RemoveRequestParameter=red
+        - RemoveResponseHeader=X-Response-Foo
 ----
 ====
 
-This will remove the `red` parameter before it is sent downstream.
+This will remove the `X-Response-Foo` header from the response before it is returned to the gateway client.
+
+To remove any kind of sensitive header, you should configure this filter for any routes for which you may want to do so.
+In addition, you can configure this filter once by using `spring.cloud.gateway.default-filters` and have it applied to all routes.
 
-=== `RequestHeaderSize` `GatewayFilter` Factory
+
+=== The `RequestHeaderSize` `GatewayFilter` Factory
 
 The `RequestHeaderSize` `GatewayFilter` factory takes `maxSize` and `errorHeaderName` parameters.
-The `maxSize` parameter is the maximum data size allowed of the request header (incuding key and value). The `errorHeaderName` parameter sets the name of the response header containing an error message, by default it is "errorMessage".
+The `maxSize` parameter is the maximum data size allowed of the request header (including key and value). The `errorHeaderName` parameter sets the name of the response header containing an error message, by default it is "errorMessage".
```

Comment:
Change `of the request header` to `by the request header` (`of` to `by`).

I know you didn't make that error. Thanks for fixing it anyway.